openpype/settings/defaults/system_settings/tools.json

```diff
@@ -1,90 +1,403 @@
+            },
+            "variants": {
+                "6-10-01": {
+                    "host_names": [
+                        "maya"
                     ],
+                    "app_variants": [],
                     "environment": {
-                        "RFMTREE": {
-                            "windows": "C:\\Program Files\\Pixar\\RenderManForMaya-24.3",
-                            "darwin": "/Applications/Pixar/RenderManForMaya-24.3",
-                            "linux": "/opt/pixar/RenderManForMaya-24.3"
-                        },
-                        "RMANTREE": {
-                            "windows": "C:\\Program Files\\Pixar\\RenderManProServer-24.3",
-                            "darwin": "/Applications/Pixar/RenderManProServer-24.3",
-                            "linux": "/opt/pixar/RenderManProServer-24.3"
-                        }
+                        "VRAY_VERSION": "6.10.01"
                     }
+                }
+            }
+        },
+        "vraynuke": {
+            "environment": {
+                "VRAY_FOR_NUKE_13_0_PLUGINS": {
+                    "windows": "{STUDIO_SW}/APP/VRAYNUKE/{VRAYNUKE_VERSION}/NUKE{NUKE_VRAY_VERSION}/WINDOWS/nuke_vray/plugins/vray"
+                },
+                "NUKE_PATH": {
+                    "windows": [
+                        "{STUDIO_SW}/APP/VRAYNUKE/{VRAYNUKE_VERSION}/NUKE{NUKE_VRAY_VERSION}/WINDOWS/nuke_root",
+                        "{NUKE_PATH}"
+                    ]
+                },
+                "PATH": {
+                    "windows": [
+                        "{STUDIO_SW}/APP/VRAYNUKE/{VRAYNUKE_VERSION}/NUKE{NUKE_VRAY_VERSION}/WINDOWS/nuke_vray",
+                        "{PATH}"
+                    ]
                 },
-                "__dynamic_keys_labels__": {
-                    "24-3-maya": "24.3 RFM"
+                "VRAY_AUTH_CLIENT_FILE_PATH": "{STUDIO_SW}/APP/VRAY"
+            },
+            "variants": {
+                "5-20-00": {
+                    "host_names": [
+                        "nuke"
+                    ],
+                    "app_variants": [],
+                    "environment": {
+                        "VRAYNUKE_VERSION": "5.20.00"
+                    }
                 }
             }
         },
         "__dynamic_keys_labels__": {
-            "mtoa": "Autodesk Arnold",
-            "vray": "Chaos Group Vray",
-            "yeti": "Peregrine Labs Yeti",
-            "renderman": "Pixar Renderman"
+            "htoa": "Arnold for Houdini (example)",
+            "mtoa": "Arnold for Maya (example)",
+            "rendermanMaya": "Renderman for Maya (example)",
+            "yetiMaya": "Yeti for Maya (example)",
+            "redshiftMaya": "Redshift for Maya (example)",
+            "redshift3dsmax": "Redshift for 3dsmax (example)",
+            "mGear": "mGear for Maya (example)",
+            "vrayMaya": "Vray for Maya (example)",
+            "vraynuke": "Vray for Nuke (example)"
```

Comment:
Why do the maya equivalents have a capital `M` but the Nuke one doesn't? `vraynuke` versus `vrayMaya`?

Reply:
That's not actually being touched here at the end. So resolving conversations....to answer though. I'd say just an accident.